content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/ios.md

```diff
@@ -268,99 +268,99 @@ Datadog.setUserInfo(id: "1234", name: "John Doe", email: "john@doe.com")
+`service`
+: The service name associated with data sent to Datadog. The default value is set to the application bundle identifier.
+
+`site`
+: Sets the Datadog server endpoint that data is sent to. The default value is `.us1`.
+
+`uploadFrequency`
+: Sets the preferred frequency of uploading data to Datadog. Available values include: `.frequent`, `.average`, and `.rare`.
 
 ### RUM configuration
 
 You can use the following properties in `RUM.Configuration` when enabling RUM:
 
-`applicationID`
-: The RUM application identifier.
-
-`sessionSampleRate`
-: Sets the sampling rate for RUM sessions. The `sessionSampleRate` value must be between `0.0` and `100.0`. A value of `0.0` means no sessions are sent, `100.0` means all sessions are sent to Datadog. If not configured, the default value of `100.0` is used.
+`actionEventMapper`
+: Sets the data scrubbing callback for actions. This can be used to modify or drop action events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
 
-`uiKitViewsPredicate`
-: Enables tracking `UIViewControllers` as RUM views. You can use default implementation of `predicate` by setting the `DefaultUIKitRUMViewsPredicate` or implement [your own `UIKitRUMViewsPredicate`](#automatically-track-views) customized for your app.
+`appHangThreshold`
+: Sets the threshold for reporting when an app hangs. The minimum allowed value for this option is `0.1` seconds. To disable app hangs reporting, set this to `nil`. For more information, see [Add app hang reporting][10].
 
-`uiKitActionsPredicate`
-: Enables tracking user interactions (taps) as RUM actions. You can use the default implementation of `predicate` by setting the `DefaultUIKitRUMActionsPredicate` or implement [your own `UIKitRUMActionsPredicate`](#automatically-track-user-actions) customized for your app.
+`applicationID`
+: The RUM application identifier.
 
-`urlSessionTracking`
-: Enables tracking `URLSession` tasks (network requests) as RUM resources. The `firstPartyHostsTracing` parameter defines hosts that are categorized as `first-party` resources (if RUM is enabled) and have tracing information injected (if tracing feature is enabled). The `resourceAttributesProvider` parameter defines a closure to provide custom attributes for intercepted resources that is called for each resource collected by the RUM iOS SDK. This closure is called with task information and may return custom resource attributes or `nil` if no attributes should be attached.
+`customEndpoint`
+: A custom server URL for sending RUM data.
 
-`trackFrustrations`
-: Determines whether automatic tracking of user frustrations should be enabled. By default, this is set to `true`.
+`errorEventMapper`
+: Sets the data scrubbing callback for errors. This can be used to modify or drop error events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
```

Comment:
```suggestion
: The data scrubbing callback for errors. This can be used to modify or drop error events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
```